workspaces/sonarqube/plugins/sonarqube/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@backstage-community/plugin-sonarqube",
-  "version": "0.8.1",
+  "version": "0.8.0",
```

Comment:
```suggestion
  "version": "0.8.1",
```

We don't want to change this, the release process handles all the versioning for us 👍